.zenodo.json

```diff
@@ -315,6 +321,9 @@
       "affiliation": "National Technical University of Athens, Greece",
       "name": "Raktivan, Konstantinos"
     },
+    {
+      "name": "Cal\u00e1bkov\u00e1, Mark\u00e9ta"
```

Comment:
@MeggyCal I've added you to the Zenodo fragment, and you will be listed as an author in the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). Please let me know if you would like your name to be rendered otherwise, or if you have an affiliation and/or [ORCID](https://orcid.org/) you would to use. I can also remove you if you'd rather not be listed as an author.

Reply:
thanks a lot, I am honoured :) it is good as it is, thank you for messing up with unicode characters :)